app/views/docs/getting-started-for-android.phtml

```diff
@@ -0,0 +1,126 @@
+
+<div class="ide" data-lang="html" data-lang-label="XML">
+    <pre class="line-numbers"><code class="prism language-xml" data-prism><?php echo $this->escape('<activity android:name="io.appwrite.views.CallbackActivity" >
+        <intent-filter android:label="android_web_auth">
+            <action android:name="android.intent.action.VIEW" />
+            <category android:name="android.intent.category.DEFAULT" />
+            <category android:name="android.intent.category.BROWSABLE" />
+            <data android:scheme="appwrite-callback-[PROJECT_ID]"  />
+        </intent-filter>
+    </activity>
+'); ?>
+</code></pre>
+</div>
+
+<h2>Init your SDK</h2>
+
+<p>Initialize your SDK code with your project ID, which can be found in your project settings page.</p>
+
+<div class="ide" data-lang="kotlin" data-lang-label="Android SDK">
+    <pre class="line-numbers"><code class="prism language-kotlin" data-prism>import io.appwrite.Client
+import io.appwrite.services.Account
+
+val client = Client(context)
+  .setEndpoint("https://[HOSTNAME_OR_IP]/v1") // Your API Endpoint
+  .setProject("5df5acd0d48c2") // Your project ID
+  .setKey("919c2d18fb5d4...a2ae413da83346ad2") // Your secret API key
+  .setSelfSigned() // For self signed certificates only use for development
+</code></pre>
+</div>
+
+<p>Before starting to send any API calls to your new Appwrite instance, make sure your Android emulators has network access to the Appwrite server hostname or IP address.</p>
+
+<p>When trying to connect to Appwrite from an emulator or a mobile device, localhost is the hostname for the device or emulator and not your local Appwrite instance. You should replace localhost with your private IP as the Appwrite endpoint's hostname. You can also use a service like <a href="https://ngrok.com/" target="_blank" rel="noopener">ngrok</a> to proxy the Appwrite API.</p>
+
+<h2>Make Your First Request</h2>
+
+<p>Once your SDK object is set, access any of the Appwrite services and choose any request to send. Full documentation for any service method you would like to use can be found in your SDK documentation or in the API References section.</p>
+
+<div class="ide" data-lang="kotlin" data-lang-label="Android SDK">
+                    <pre class="line-numbers"><code class="prism language-kotlin" data-prism>// Register User
+val account = Account(client)
+val response = account.create("email@example.com", "password")
+val json = response.body?.string()
+</code></pre>
+</div>
+
+<h2>Full Example</h2>
+<div class="ide" data-lang="dart" data-lang-label="Dart SDK">
```

Comment:
You might wanna replace references to dart here

Reply:
done